src/workflows/airqo_etl_utils/satellilte_utils.py

```
@@ -0,0 +1,160 @@
+
+        train_target, validation_target, test_target = (
+            train_data[target_col],
+            validation_data[target_col],
+            test_data[target_col],
+        )
+
+        sampler = optuna.samplers.TPESampler()
+        pruner = optuna.pruners.SuccessiveHalvingPruner(
+            min_resource=10, reduction_factor=2, min_early_stopping_rate=0
+        )
+        study = optuna.create_study(
```

Ruff

Local variable `study` is assigned to but never used

Remove assignment to unused variable `study`

(F841)